src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -121,12 +129,13 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="-lrt" Condition="'$(TargetOS)' != 'osx'" />
       <LinkerArg Include="-licucore" Condition="'$(TargetOS)' == 'osx'" />
       <LinkerArg Include="-L/usr/lib/swift" Condition="'$(TargetOS)' == 'osx'" />
+      <LinkerArg Include="@(StaticNumaLibs)" Condition="'$(StaticNumaLinking)' == 'true'" />
       <LinkerArg Include="@(StaticICULibs)" Condition="'$(StaticICULinking)' == 'true'" />
       <LinkerArg Include="@(StaticSslLibs)" Condition="'$(StaticOpenSslLinking)' == 'true'" />
       <LinkerArg Include="-dynamiclib" Condition="'$(TargetOS)' == 'osx' and '$(NativeLib)' == 'Shared'" />
       <LinkerArg Include="-shared" Condition="'$(TargetOS)' != 'osx' and '$(NativeLib)' == 'Shared'" />
       <!-- binskim warning BA3001 PIE disabled on executable -->
-      <LinkerArg Include="-pie" Condition="'$(TargetOS)' != 'osx' and '$(NativeLib)' == '' and '$(PositionIndependentExecutable)' != 'false'" />
+      <LinkerArg Include="-pie" Condition="'$(TargetOS)' != 'osx' and '$(NativeLib)' == '' and '$(StaticExecutable)' != 'true' and '$(PositionIndependentExecutable)' != 'false'" />
```

Comment:
This is to avoid warning with `-p:StaticExecutable=true`:

```
clang-15 : warning : argument unused during compilation: '-pie' [-Wunused-command-line-argument] 
```

Reply:
Does it mean that address space randomization is not available with fully statically linked binaries?

Reply:
clang 9 and gcc 8 added `-static-pie` option, which results in:
```sh
$ file bin/Debug/net8.0/linux-musl-arm64/publish/exe1 | fold

bin/Debug/net8.0/linux-musl-arm64/publish/exe1: ELF 64-bit LSB pie executable, A
RM aarch64, version 1 (SYSV), static-pie linked, BuildID[sha1]=b70d838c9e3ec616f
e5d842514120cf2145ff76c, with debug_info, not stripped
```
I will add it below.